### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Aryalfrat**
update readme

### CODE REVIEW

**Review by fenrir-z — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -791,44 +794,15 @@ Mining and inference mirrors The configuration file templates for `industryessen
...
 ## 7.2. License
 
 YMIR is licensed under version 2.0 of the Apache License. See the [LICENSE](https://github.com/IndustryEssentials/ymir/blob/master/LICENSE) file for details.
 
 ## 7.3. Contact us
 
-Contact us with further questions：contact.viesc@gmail.com
+Contact us with further questions:contact.viesc@gmail.com
+
+Or join our SLACK, we will answer your questions in real time: https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q
```
Comment (under-chaos) [inline README.md]: Join our [Slack community](https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q) to chat to our engineers about your use cases, questions, and support queries.

Comment (under-chaos) [reply README.md]: `Join our [Slack community](https://join.slack.com/t/ymir-users/shared_invite/zt-ywephyib-ccghwp8vrd58d3u6zwtG3Q) to chat to our engineers about your use cases, questions, and support queries.`

**Review by phoenix-xhuang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -791,44 +794,15 @@ Mining and inference mirrors The configuration file templates for `industryessen
 
 After successful training, the system will output the ID of the model. The user can find the corresponding file according to this id at `--model-location`. In fact, it is a tar file that can be extracted directly using the tar command to get the "mxnet" model file in parameters and JSON format.
 
-**How to view and modify the label file of the category name?**
```
Comment (phoenix-xhuang) [inline README.md:794]: 这里把@智维 刚加的内容删掉了

File: README_zh-CN.md [hunk 3]
```diff
@@ -765,44 +769,15 @@ YMIR repo中的任何代码都应遵循编码标准，并将在CI测试中进行
 
 成功完成训练后，系统会输出模型的 id，用户可以根据这个 id 到 `--model-location` 位置找到对应的文件，它事实上是一个 tar 文件，可以直接使用 tar 命令解压，得到 params 和 json 格式的 mxnet 模型文件。
 
-*  如何查看及修改类别名称标签文件？
```
Comment (phoenix-xhuang) [inline README_zh-CN.md:768]: 这里也是

**Review by fenrir-z — APPROVED** [review]

**Review by under-chaos — APPROVED** [review]

**Pull request merged**